client/packages/programs/src/JsonForms/common/components/Label.tsx

```diff
@@ -12,7 +12,7 @@ import { FORM_LABEL_COLUMN_WIDTH } from '../styleConstants';
 
 export const labelTester = rankWith(3, uiTypeIs('Label'));
 
-type LabelVariant = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p';
+type LabelVariant = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'p' | 'h1Modal';
```

Comment:
This doesn't seem to be the cleanest way to do it? e.g. should the label really care in which parent it is rendered?

Can't the parent take care of how to render the label? for example could we assign a css class instead and set the styles in the container?

Reply:
Just reading through the issue, we can also just have a generic header for the enrolment forms, i.e. the header with "Enrolment Form" and enrolment name followed by the json forms component